doc/website/release-notes/iceoryx-unreleased.md

```diff
@@ -165,6 +165,7 @@
 - Roudi Environment independent from Googletest [#1533](https://github.com/eclipse-iceoryx/iceoryx/issues/1533)
 - Move test class for ctor and assignment operators to hoofs testing [#2041](https://github.com/eclipse-iceoryx/iceoryx/issues/2041)
 - Refactor `FixdePositionContainer` and move to `dust` [#2044](https://github.com/eclipse-iceoryx/iceoryx/issues/2044)
+- Cleanup or Remove ObjectPool [#66](https://github.com/eclipse-iceoryx/iceoryx/issues/66)
```

Comment:
Do we usually name the issue title here, or what has been done? It reads strange because "cleanup" and "remove" are contrary.

Reply:
Yes, at least I usually copy the issue description ... not sure if everybody does the same